### ISSUE TITLE
IOS EXC_BAD_ACCESS (code=1, address=0x10) while try to load local file 

### ISSUE THREAD

**Issue by vanLesson** [Alwinator/react-native-http-bridge-refurbished#8]
I am trying to get a local file from the system RNFile ( file = await RNFS.readFile(cacheFile))
 and send the file using, res.send(200, type, file); but the program crashed due to this error
 <img width="778" alt="image" src="https://user-images.githubusercontent.com/55431955/223140340-b410227d-092c-4da9-bf04-eb726a1302ee.png">
then I'm trying to add this condition, my app loaded HTML but the additional file doesn't  load 
<img width="778" alt="image" src="https://user-images.githubusercontent.com/55431955/223141150-7ac4cc1a-8a9e-4744-b66e-df4f249bc85b.png">

**Comment by Alwinator** [comment]
@vanLesson Usually this error indicates that you are trying to access a pointer that is invalid or no longer exists, but not sure what the problem is in this case. Can you please provide more details so I can investigate the problem?

- Does loading the file alone work? (Without sending it with res.send)
- Which versions are you using React, react-native-http-bridge-refurbished, ...
- Please provide a minimal reproduction project which includes the bug.

**Comment by vanLesson** [comment]
Thanks for the answer, I solved this problem, with each request, the _completionBlocks referred to the missing area in memory: this is why I moved their cleanup for each request
```RCT_EXPORT_METHOD(start:(NSInteger) port
                  serviceName:(NSString *) serviceName)
{
    RCTLogInfo(@"Running HTTP bridge server: %ld", port);

    _completionBlocks = [[NSMutableDictionary alloc] init];

    dispatch_sync(dispatch_get_main_queue(), ^{
        _webServer = [[GCDWebServer alloc] init];
        
        [self initResponseReceivedFor:_webServer forType:@"POST"];
        [self initResponseReceivedFor:_webServer forType:@"PUT"];
        [self initResponseReceivedFor:_webServer forType:@"GET"];
        [self initResponseReceivedFor:_webServer forType:@"DELETE"];
        
        [_webServer startWithPort:port bonjourName:serviceName];
    });
}```

**Comment by Alwinator** [comment]
Thank you very much I would really appreciate if you could create a pull request for that ;)

**Comment by vanLesson** [comment]
I created a pool request, this commit only fixes this error, but I also added the ability to send files in byte map format for pictures, if it's interesting, I'll create another pool request :)

**Comment by Alwinator** [comment]
@vanLesson Thank you, I have just merged it! The fix is available in 1.2.4. Please also add the other feature, I think it is a useful feature! :)
